packages/admin-stories/src/admin/error-handling/ErrorDialogGraphql.tsx

```diff
@@ -0,0 +1,86 @@
+import { boolean, text } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as React from "react";
+
+import { apolloSwapiStoryDecorator } from "../../apollo-swapi-story.decorator";
+import { errorDialogStoryProviderDecorator } from "../../error-dialog-provider.decorator";
+
+interface People {
+    id: string;
+    name: string;
+    birthYear: string;
+    gender: string;
+    homeworld: {
+        name: string;
+    };
+}
+interface QueryData {
+    allPeople: {
+        people: People[];
+    };
+}
+
+const Story: React.FunctionComponent = () => {
+    const query = gql`
+        ${text("GQL Query", "query StarWarsPeople {allPeople { people { id name birthYear gender homeworld{ name } } }}")}
+    `;
+
+    const globalErrorScope = boolean("Use Error Dialog", true);
+
+    const { tableData, api, loading, error } = useTableQuery<QueryData, {}>()(query, {
+        resolveTableData: (data) => ({
+            data: data.allPeople.people,
+            totalCount: data.allPeople.people.length,
+        }),
+        context: globalErrorScope ? undefined : LocalErrorScopeApolloContext,
+    });
+
+    return (
+        <div>
+            <Alert severity={"info"}>
+                <Typography paragraph={true}>Error Dialog Provider automatically shows an Error Dialog if an GQL Error happens. </Typography>
+                <Typography paragraph={true}>
+                    Go to knobs and try to make an invalid gql query. As soon you get an GQL Error the error dialog should appear.
+                </Typography>
+            </Alert>
+            <TableQuery api={api} loading={loading} error={globalErrorScope ? undefined : error}>
+                {tableData && (
+                    <Table
+                        {...tableData}
+                        columns={[
+                            {
+                                name: "id",
+                                header: "Id",
+                            },
+                            {
+                                name: "name",
+                                header: "Name",
+                            },
+                            {
+                                name: "birthYear",
+                                header: "Birthyear",
+                            },
+                            {
+                                name: "gender",
+                                header: "Gender",
+                            },
+                            {
+                                name: "homeworld.name",
+                                header: "Homeworld",
+                            },
+                        ]}
+                    />
+                )}
+            </TableQuery>
+        </div>
+    );
+};
+
+storiesOf("@comet/admin/error-handling/error-dialog", module)
+    .addDecorator(apolloSwapiStoryDecorator())
+    .addDecorator(errorDialogStoryProviderDecorator())
```

Comment:
don't you think that such a decorator is too much magic for a story that mainly is used as documentation?

Or is this not an issue because the starter will have the provider and a developer doen't have to take care about it

Reply:
i think that is ok so. The focus should be, how to developer can use the error dialog.  For more clearance of setting up this error provider, i added a story of setting up the error dialog provider to the documentation area.